lib/hackney/income/tenancy_prioritiser/universal_housing_criteria.rb

```diff
@@ -88,6 +97,18 @@ def balance
           attributes.fetch(:current_balance).to_f
         end
 
+        def nosp_served_date
+          return nil if attributes.dig(:nosp_served_date).nil?
```

Comment:
`dig` with one argument can just be hash access, if you want. Also, I think `nil` is returned implicitly if you don't specify a return value?

```suggestion
          return if attributes[:nosp_served_date].nil?
```